.travis.yml

```diff
@@ -49,8 +52,8 @@ matrix:
         - env: DJANGO_VERSION=">=1.9,<1.10"
         - python: "pypy"
 
-services:
-    - elasticsearch
+#services:
```

Comment:
We need to keep testing with 1.x. I'm not sure whether it'd be easier to run the tests with an alternate port or to use a build matrix to pick one of the two. It looks like https://github.com/travis-ci/apt-source-whitelist/commit/189b706dc54ccbdd99fae5489fcbc71ee1a918c5 was merged awhile back so I think we could use a matrix like what they're doing here:

https://github.com/insomnia-lab/libreant/pull/263/commits/49fc5d06d6bed5153f85ffb3e1bfd4903575df9b